drone/helper.go

```diff
@@ -10,6 +10,19 @@ import (
 	// "time"
 )
 
+func convertWindowsPath(str string) string {
+	// Split path into Array
+	var pwds = strings.Split(str, "\\")
+
+	// Convert drive "C:" to "/c"
+	rp := regexp.MustCompile("(^[a-zA-Z]):")
+	pwds[0] = rp.ReplaceAllString(pwds[0], "/$1")
```

Comment:
we can use https://golang.org/pkg/path/filepath/#ToSlash here which will handle the path separator conversion